Controller/Adminhtml/Import/Ajax.php

```diff
@@ -48,19 +48,29 @@ public function __construct(
     public function execute()
     {
         $storeId = (int)$this->request->getParam('storeId');
+        $time = time();
```

Comment:
What happens with this time variable if the import takes 1 hour? Would it ever change? It should be accurate for every backend call.

Reply:
No, it will not change. I was wondering if it would be better to place it inside the backendCall where the actual call to the api is happening. I will remove it from the Ajax controller and place it in the request function.